01_login.php

```diff
@@ -54,34 +56,77 @@
 		<div class="w3-top">
 			<div class="w3-bar w3-white w3-wide w3-padding w3-card">
 				<img src="images/placeholder.jpg" width="40" height="40">
-				<a href="indexV1.php" class="w3-bar-item w3-button"><b>logo</b> </a>
```

Comment:
deleted href button as there was no reason to have this in the login page, users wont be able to use it due to not being logged in anyway.